src/SILHS/version_silhs.txt

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
This file never appears in the clubb repo, and so we can simply ignore it.

Reply:
@vlarson : Just for your reference. This file is changed to resolve merge conflicts. It is not related to the convergence test simulations